Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -802,20 +802,31 @@ class EndIf: ControlFlowOperation {
     }
 }
 
-/// The block content is the body of the default case
+/// The first input is the operand to the switch block
+/// The second optional input is the operand to the first swtich case. When the second input is not defined, this generates a default switch case.
+/// The block content is the body of the first switch case
 class BeginSwitch: ControlFlowOperation {
-    init() {
-        super.init(numInputs: 1, attributes: [.isBlockBegin])
+
+    var firstCaseIsDefaultCase: Bool {
+        return numInputs == 1
+    }
+
+    init(numArguments: Int) {
+        super.init(numInputs: numArguments, attributes: [.isBlockBegin])
     }
 }
 
 class BeginSwitchCase: ControlFlowOperation {
     /// If true, causes the preceding case to fall through to it (and so no "break;" is emitted by the Lifter)
     let fallsThrough: Bool
```

Comment:
Should we call this `previousCaseFallsThrough` to make it more clear what this means?